pkgs/build-support/vm/linuxkit-builder/default-key.pub

```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCi0IEOV7nLjLaLtu7eRAJYMe2EeqS90KDozO3KzuRcvzlZ/DJkFHuLI0PutlWjDq/I2L7i76jm7DkpVuKW9k2CsFOYlM5/Nrg4YtEgFA7zOSI1Wry8jITvauF6sBFVYNaN9H6Aa80enB9+ysKsU3vamjQcs5USxU/g1PjxmgFMBqNeQ72hGUQJeIh+PaKG3O2r61VFe/XX7GTe+wdrAz7g5XNrzhMV9WdGVldanvtnHuz6UZ1s+/tqznEnMKzm2P2XCNdU4eafBsfJvnVT4FxR4rpRRkj5eVer5PouA03vuAb7Zx5ogdYfBeNQLGytMEbzUfMwKCop9i2rH4qKd+l5
```

Comment:
Not sure if worth doing this, but you can derive a pubkey from a privkey with ssh-keygen, so it could save the amount of obscure cryptononsense in the repo.